docs/OncoTree-Mapping-Tool.md

```diff
@@ -184,3 +188,26 @@ After filling in the `ONCOTREE_CODE` column with an OncoTree code for each sampl
 |P16|S16|65|MNM|
 |P17|S17|66|MYCF|
 |P18|S18|66|RBL|
+
+# Ontology Mapping Tool
+
+The Ontology Mapping tool was developed to facilitate the mapping between different cancer classification systems. We currently allow the mappings between OncoTree, ICD-O, NCIt, UMLS and HemeOnc systems. The tool and the mapping file can be found [here](https://github.com/cBioPortal/oncotree/tree/master/scripts)
+
+### Running the tool
+
+The tool can be run with the following command:
+```
+python <path/to/scripts/ontology_mapping_tool.py> --source-file <path/to/source/file> --target-file <path/to/target/file> --target-code <target_ontology_code>
+```
+
+**Options**
+```
+ -i | --source-file: This is the source file path. The source file must contain one of the ONCOTREE_CODE, ICDO_CODE, NCIT_CODE, UMLS_CODE or HEMEONC_CODE in the file header and it must contain codes corresponding to the Ontology System.
```

Comment:
What happens if the source file contains 2 or 3 of these fields? We should know what the script behavior should be and let users know here. Is there a preset order of preference? Or maybe we need a fourth command line argument "-s | --source-code" ?

Reply:
@sheridancbio for the time being we are only allowing one source code. The input file contains the source code. We can modify here to say just add one source code. @rmadupuri can this be updated in the read me?

Reply:
@sheridancbio, since we are currently allowing only one to one ontology mapping, if the source file has 2 or 3 of these fields the user will be asked to enter one ontology to map by.

Reply:
Ok @rmadupuri ... so the script will exit / fail and tell the user that only one ontology column can be present in the source file?

Reply:
Yes @sheridancbio, just added another comment pointing to the block in code that handles this case.